api/cluster/pdb.go

```diff
@@ -90,7 +89,7 @@ func (c *controller) deployPodDisruptionBudget(ctx context.Context, pdb *PodDisr
 	pdbCfg.WithLabels(pdb.Labels)
 	pdbCfg.WithSpec(pdbSpec)
 
-	_, err = c.policyClient.PodDisruptionBudgets(pdb.Namespace).Apply(ctx, pdbCfg, metav1.ApplyOptions{})
+	_, err = c.policyClient.PodDisruptionBudgets(pdb.Namespace).Apply(ctx, pdbCfg, metav1.ApplyOptions{FieldManager: "application/apply-patch"})
```

Comment:
Missed this in the previous PR. But what's the reason for using `Apply` instead of `Create`? https://github.com/kubernetes/client-go/blob/v0.27.3/kubernetes/typed/policy/v1/poddisruptionbudget.go#L118

Reply:
So we can use Apply for both Undeploy and Redeploy. Otherwise, we need to implement a function that call Update.

Reply:
> So we can use Apply for both Undeploy and Redeploy

Do you mean for both Deploy and Redeploy?

Reply:
Oops, yes, you're right. It's for Deploy and Redeploy.